mmv1/products/dlp/DiscoveryConfig.yaml

```diff
@@ -485,6 +491,128 @@ properties:
             # The fields below are necessary to include the "secretsDiscoveryTarget" target in the payload
           send_empty_value: true
           allow_empty_object: true
+        - !ruby/object:Api::Type::NestedObject
+          name: cloudStorageTarget
```

Comment:
To confirm, all of these field are mutable?

Reply:
Yes, they are mutable.